cpp/src/Ice/LoggerI.cpp

```diff
@@ -25,15 +25,15 @@ namespace
     const chrono::minutes retryTimeout = chrono::minutes(5);
 }
 
-Ice::LoggerI::LoggerI(const string& prefix, const string& file, bool convert, size_t sizeMax)
-    : _prefix(prefix),
+Ice::LoggerI::LoggerI(string prefix, const string& file, bool convert, size_t sizeMax)
```

Comment:
Shouldn't file be a string too?

Reply:
Fixed. The logic here was to update the public API but leave internal uses of const string& alone.